src/main/java/org/apache/commons/text/beta/StrTokenizer.java

```diff
@@ -455,7 +455,7 @@ public String previousToken() {
      *
      * @return this, to enable chaining
      */
-    public org.apache.commons.text.StrTokenizer reset() {
+    public org.apache.commons.text.beta.StrTokenizer reset() {
```

Comment:
This actually is a great change to make to catch all of my mistakes in leaving the full qualified package when I moved this stuff over from [lang]